Client/Frontend/Shields/ShareTrackersController.swift

```diff
@@ -176,11 +155,13 @@ private class ShareTrackersView: UIView, Themeable {
         $0.backgroundColor = .clear
         $0.setContentCompressionResistancePriority(.required, for: .horizontal)
         $0.numberOfLines = 0
+        $0.textColor = .white
```

Comment:
This view should have non dynamic colors?

Reply:
Should be OK because it uses a gradient bg